tests/worker/ingestion/team-manager.test.ts

```diff
@@ -157,22 +154,23 @@ describe('TeamManager()', () => {
             teamManager.teamCache.clear()
 
             const team = await teamManager.fetchTeam(2)
-            expect(team?.event_names).toEqual(['$pageview', 'new-event'])
-            expect(team?.event_names_with_usage).toEqual([
-                { event: '$pageview', usage_count: 2, volume: 3 },
-                { event: 'new-event', usage_count: null, volume: null },
-            ])
-            expect(team?.event_properties).toEqual(['property_name', 'numeric_prop', 'number'])
-            expect(team?.event_properties_with_usage).toEqual([
-                { key: 'property_name', usage_count: null, volume: null },
-                { key: 'numeric_prop', usage_count: null, volume: null },
-                { key: 'number', usage_count: null, volume: null },
-            ])
-            expect(team?.event_properties_numerical).toEqual(['numeric_prop', 'number'])
+            // expect(team?.event_names).toEqual(['$pageview', 'new-event'])
+            // expect(team?.event_names_with_usage).toEqual([
+            //     { event: '$pageview', usage_count: 2, volume: 3 },
+            //     { event: 'new-event', usage_count: null, volume: null },
+            // ])
+            // expect(team?.event_properties).toEqual(['property_name', 'numeric_prop', 'number'])
+            // expect(team?.event_properties_with_usage).toEqual([
+            //     { key: 'property_name', usage_count: null, volume: null },
+            //     { key: 'numeric_prop', usage_count: null, volume: null },
+            //     { key: 'number', usage_count: null, volume: null },
+            // ])
+            // expect(team?.event_properties_numerical).toEqual(['numeric_prop', 'number'])
```

Comment:
Shouldn't have commented out code in tests I think?

Reply:
ooh, how did I forget that in :(